src/h5web/App.module.css

```diff
@@ -8,16 +8,35 @@
   background-color: var(--primary-bg);
 }
 
-.dataVisualizer {
+.mainArea {
   display: flex;
-  scrollbar-width: thin;
+  flex-direction: column;
 }
 
-.metadataViewer {
-  overflow: scroll !important;
-  scrollbar-width: thin;
+.roleToggler {
+  display: flex;
+  justify-content: center;
```

Comment:
`Inspect` and `Visualize` buttons are centered for now but will be put to `flex-end` when adding the breadcrumbs